packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -115,6 +116,13 @@ const start = async (zcf, privateArgs) => {
   } = /** @type { Terms & AMMTerms } */ (zcf.getTerms());
   assertIssuerKeywords(zcf, ['Central']);
   assert(centralBrand !== undefined, X`centralBrand must be present`);
+
+  const centralDisplayInfo = await E(centralBrand).getDisplayInfo();
```

Comment:
This `await` adds a round-trip to contract startup. That could be avoided by combining it using `Promise.all` with `const paramManager = await makeParamManager(` below, at the cost of some code clarity.

@Chris-Hibbert I'm curious what you think.

Reply:
My usual rule of thumb is to not worry about performance issues (particularly one-time start up costs) until measurement shows them to be a problem, but I think you've been actually measuring start up times, so anyway.

I don't have a problem with using `Promise.all()` to aggregate awaits even when they're working on different tasks. This test doesn't have to be the first thing done. It's entire purpose would be to prevent accidentally breaking the AMM, so it should never fail if the AMM would have been useful.